src/types/profile.js

```diff
@@ -357,15 +357,27 @@ export type VisualProgress = {|
  * into the geckoprofile. More information about browsertime can be found through https://github.com/mozilla/browsertime.
  */
 export type VisualMetrics = {|
-  SpeedIndex: number,
-  PerceptualSpeedIndexProgress: VisualProgress[],
-  LastVisualChange: number,
+  // ContentfulSpeedIndex and ContentfulSpeedIndexProgress generated here
+  // https://github.com/mozilla/browsertime/blob/master/vendor/visualmetrics.py#L1331
```

Comment:
Thanks for adding these. These aren't permalinks, and quickly grow stale. I would recommend going to the github page and hitting `Y` for a shortcut to get the permalink, or doing it through this menu:

<img width="358" alt="image" src="https://user-images.githubusercontent.com/1588648/64562771-465f4980-d313-11e9-8a25-ef3d9b7ece59.png">